### DISCUSSION TITLE
show chinese  text or other font

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#502] [Q&A]
the dwg file insert  a block and the block hava text include english word and chinese word 
(font name is 宋体 using microsoft  chinese system font   and font style is regular)
,and to bim file by itwin snapshot.exe, but in imdeljs open the bim file,just show the english word ,and show a empty box in the position of chinese word .and can not show chinese word .

**Comment by donhfu** [comment]
An SHX bigfont should work but a wide true type is yet to be supported.  Can you send a small test DWG file that has your font so we can be sure to get it covered?
